src/nuklear_font.c

```diff
@@ -195,7 +195,12 @@ nk_font_bake_pack(struct nk_font_baker *baker,
         } while ((it = it->n) != config_iter);
     }
     *height = 0;
-    *width = (total_glyph_count > 1000) ? 1024 : 512;
+   
+	/* Be able to handle very large fonts */
+	int factor = total_glyph_count / 1024;
```

Comment:
Maybe also comment on the origin of this equation :wink:.

Reply:
What origin? It was just a bit of guesswork. and it works to keep the display from turning all black when too big a font is loaded.

Reply:
Well, I meant the "guesswork" as you call it. I think there could be some dependency on font size etc., but don't know where to start (didn't do any analysis).

Reply:
I'd love to tell you I did some deep analysis, but it was just a "hunch" that seems to play out. I was able to load huge font at a large size, when before I could only load some of the characters at a medium size.

It fixes the specific problem I was facing ,but I don't know if there's a better solution out there.

Reply:
@ronaaron you do not seem to be the only one having [a problem with bigger fonts](https://github.com/Immediate-Mode-UI/Nuklear/issues/77#issuecomment-605383306 ), so I'd definitely like to have a reasonable solution for this. I actually think even the original width computation was not correct. But instead of making it incorrect again, I'd rather suggest trying to consult the rect packing and font library from [STB](https://github.com/nothings/stb ).